apps/src/code-studio/soundLibrary.json

```diff
@@ -1946,19 +2388,131 @@
     "animals": [
       "category_background/seagulls"
     ],
+    "app": [
+      "category_app/Perfect_App_Button_1",
+      "category_app/Perfect_App_Button_2",
+      "category_app/Perfect_App_Button_3",
+      "category_app/Perfect_App_Button_4",
+      "category_app/Perfect_App_Button_5",
+      "category_app/Perfect_Clean_App_Button_Click",
+      "category_app/Perfect_Clean_App_Button_Click_2",
+      "category_app/Perfect_Organic_Button"
+    ],
+    "app_button_1": [
+      "category_app/App_Button_1"
+    ],
```

Comment:
Future cleanup: There are quite a few of these tag lookups with just one entry where the tag and the sound name differ only by case. (This isn't a problem you introduced with this PR, it's an existing issue).  Just calling out that it'd be a nice potential improvement to not need to include these in the manifest, since they could be computed and added to this set very cheaply on the client.

Reply:
I wonder if I can tackle that improvement during this sprint. If not, perhaps the next sprint as a future cleanup as you mentioned.